SNOmodels/Snakefile

```diff
@@ -368,6 +405,34 @@ rule make_statistics:
 
         pd.concat(out_df, axis='rows').to_csv(output.stats_csv, index=None, float_format='%.3f')
 
+rule mutation_classification:
+    input:
+        "results/statistics.csv"
+    output:
+        "results/mutations_classification/{hugo}_{up_ac}_{chain}_{start}-{end}_{restraints_set}_mutation_classification.csv"
+    run:
+        df=pd.read_csv(str(input))
+        columns=list(df.columns)
+        columns.append('Evaluation')
+
+        gene_list=list(set(df["hugo"].to_list()))
+        for i in gene_list:
+            df_i=df.loc[df["hugo"]==wildcards.hugo]
+            proxi_list=df_i.loc[df_i["mutation"]=="WT","proximal_Cys"].to_list()
+            output_df=pd.DataFrame(columns=columns)
+            for prox in proxi_list:
+                df_prox_cys=df_i.loc[df_i["proximal_Cys"]==prox]
+                subset_df_prox_cys=df_prox_cys[["d_SNO-sg_CYS-sg_avg","pKa_SNO_avg","mutation","proximal_Cys"]]
+                par_WT_list=subset_df_prox_cys.loc[(subset_df_prox_cys["proximal_Cys"]==prox)&(subset_df_prox_cys["mutation"]=="WT")].values.flatten().tolist()
+                df_prox_cys['pKa_classification'] = df_i.apply(lambda x: assignment_based_on_pKa(x["pKa_SNO_avg"],par_WT_list), axis=1)
+                df_prox_cys['distance_pKa_classification'] = df_i.apply(lambda x: assignment_based_on_distance_and_pKa(x["d_SNO-sg_CYS-sg_avg"],x["pKa_SNO_avg"],par_WT_list), axis=1)
```

Comment:
what happens with WT in this case? In your output you also classify the WT cases which shouldn't be classified

Reply:
done